prow/push.sh

```diff
@@ -64,4 +64,12 @@ if [[ "${new_version}" == *-dirty ]]; then
   exit 1
 fi
 echo -e "Pushing $(color-version ${new_version}) via $(color-target //prow:release-push --platforms=@io_bazel_rules_go//go/toolchain:linux_amd64) ..." >&2
-bazel run //prow:release-push --platforms=@io_bazel_rules_go//go/toolchain:linux_amd64
+# Remove retries after https://github.com/bazelbuild/rules_docker/issues/673
+for i in {1..3}; do
+  bazel run //prow:release-push --platforms=@io_bazel_rules_go//go/toolchain:linux_amd64 && exit 0 || true
```

Comment:
nit: This could be:

```sh
if ! bazel run ... ; then
    if [[ "$i" == 3 ]]; then
        echo "Failed"
        exit 1
    fi
# ...
```

Technically `foo && succeeded || failed` is not identical to
```sh
if foo; then
    succeeded
else
    failed
fi
```

As the `|| failed` will fire if `succeded` fails, too. Not germane to this case as written but better not to go there... it's a trap

Reply:
Done